src/Umbraco.Core/Models/PagedModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Umbraco.New.Cms.Core.Models;
+
+public class PagedModel<T>
+{
+    public PagedModel(long total, IEnumerable<T> items)
+    {
+        Total = total;
+        Items = items;
+    }
+
+    public IEnumerable<T> Items { get; } = Enumerable.Empty<T>();
+
+    public long Total { get; }
```

Comment:
Might be nice with an empty constructor here, for those who are object initializer leaning 😛